crates/tracing/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+extern crate crossbeam_channel;
+extern crate rustracing;
+extern crate rustracing_jaeger;
+
+pub type Span = rustracing_jaeger::Span;
```

Comment:
Should we `shrinkwraprs` this so we can add attach custom helper functions / converters / etc?

Reply:
Cool I didn't know about shrinkwrap. Probably! For now it's just a stub that's not used for real.

Reply:
:+1: